README.md

```diff
@@ -0,0 +1,152 @@
+
+
+# 📖 Blog-app <a name="about-project"></a>
+
+The **Blog App** is crafted to function as a comprehensive blog website, offering users a seamless platform to discover a diverse catalog of articles. Its purpose is to facilitate user interaction through features such as commenting and expressing appreciation through post likes. The goal is to provide a fully functional web experience for users to engage with and enjoy the content available on the platform.
+
+## 🛠 Built With <a name="built-with">Ruby</a>
+
+### Tech Stack <a name="tech-stack"></a>
+- **Ruby**
+- **PostgreSql**
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **Ruby on Rails**
+- **Creating a data model**
+- **Functional website**
+- **Unit testing**
+- **Database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> You need the following tools be installed in your computer:
+
+> - [Rails](https://guides.rubyonrails.org/)
+> - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+> - [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+> - IDE
+> - Etc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-prefered-folder
+  
+  git clone https://github.com/bhobserver/blog-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Blog
+  
+  bundle install
+```
+
+### 💾 Database
+
+Create the databases and run migrations with:
+
+```sh
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server or rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  rspec spec/
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Burhan Uddin**
+
+- GitHub: [Burhan Uddin](https://github.com/BHObserver/)
+- Twitter: [Burhan Uddin](https://twitter.com/BurhanU14173360)
+- LinkedIn: [Burhan Uddin](https://www.linkedin.com/in/bhobserver/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name='future-features'></a>
+
+- [ ]  Validations and Model specs.
+- [ ]  Add more features to the app.
+- [ ]  Add more tests.
+- [ ]  Add more styling.
+- [ ] Add more features to the app.
+- [ ]  Add more tests.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/bhobserver/blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly drop a start ⭐️ for the [repository](https://github.com/bhobserver/blog-app.git);
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [optional] I think there is a typo here, perhaps you meant **star** instead of **start**. You can correct it if I'm right.

```suggestion

## ⭐️ Show your support <a name="support"></a>

If you like this project, kindly drop a star ⭐️ for the [repository](https://github.com/bhobserver/blog-app.git);

<p align="right">(<a href="#readme-top">back to top</a>)</p>

```